model/push_notification_test.go

```diff
@@ -17,3 +17,78 @@ func TestPushNotification(t *testing.T) {
 		t.Fatal("Ids do not match")
 	}
 }
+
+func TestPushNotificationDeviceId(t *testing.T) {
+
+	msg := PushNotification{Platform: "test"}
+
+	msg.SetDeviceIdAndPlatform("android:12345")
```

Comment:
The platform doesn't matter anymore.  It can be set to anything.  It only matters in the push proxy.